src/style.css

```
@@ -0,0 +1,108 @@
+@import "../../layout_stars/src/style.css";
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 406px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
```

add padding-top and padding-bottom too
```suggestion
  padding: 32px 16px;
```